zerorpc/cli.py

```diff
@@ -245,22 +245,22 @@ def run_client(args):
     if not args.command:
         (longest_name_len, detailled_methods, service) = zerorpc_inspect(client,
                 long_doc=False, include_argspec=args.inspect)
-        print('[{0}]'.format(service))
+        print('[{0}]'.format(service), file=sys.stderr)
         if args.inspect:
             for (name, doc) in detailled_methods:
                 print(name)
         else:
             for (name, doc) in detailled_methods:
-                print('{0} {1}'.format(name.ljust(longest_name_len), doc))
+                print('{0} {1}'.format(name.ljust(longest_name_len), doc), file=sys.stderr)
         return
     if args.inspect:
         (longest_name_len, detailled_methods, service) = zerorpc_inspect(client,
                 method=args.command)
         if detailled_methods:
             (name, doc) = detailled_methods[0]
-            print('[{0}]\n{1}\n\n{2}\n'.format(service, name, doc))
+            print('[{0}]\n{1}\n\n{2}\n'.format(service, name, doc), file=sys.stderr)
```

Comment:
and again here, we are asking to inspect a single method, this is not logging, it was explicitly requested.

Reply:
I have removed this change.